cypress/support/index.ts

```diff
@@ -18,3 +18,5 @@ import './commands';
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+Cypress.on('uncaught:exception', () => false);
```

Comment:
What is this for? If this is silencing uncaught exceptions, that this seems like it's a bad thing to do, since we don't want these to slip past CI.

Reply:
ok, yeah this was a workaround for a previous error already solved